### DISCUSSION TITLE
Error opening mesh file

### DISCUSSION THREAD

**Discussion by DYLDYLDYL** [idaholab/moose#19652] [Q&A Meshing] [answered]
[01.zip](https://github.com/idaholab/moose/files/7727447/01.zip)
![2021-12-16 20-53-03屏幕截图](https://user-images.githubusercontent.com/91199452/146376808-1b38cb2f-6145-40a4-b07e-d17af162f0d2.png)

**Comment by GiudGiud** [comment, marked as answer]
Hello

The message is clear. The node set specified in the boundary parameter of the BC named bottom does not exist on the mesh. You need to add it to your mesh, either using trelis as you mentioned in your unprompted email, or using a mesh generator
https://mooseframework.inl.gov/syntax/Mesh/index.html

Guillaume

**Reply by DYLDYLDYL** [reply]
Thanks！
